dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPIImpl.java

```diff
@@ -295,7 +295,7 @@ public boolean fileNameExists(final Host host, final Folder folder, final String
 		final Identifier folderId = APILocator.getIdentifierAPI().find(folder);
 		final String path = folder.getInode().equals(FolderAPI.SYSTEM_FOLDER) ? StringPool.FORWARD_SLASH + fileName : folderId.getPath() + fileName;
 		final Identifier fileAssetIdentifier = APILocator.getIdentifierAPI().find(host, path);
-		if(( null == fileAssetIdentifier || !InodeUtils.isSet(fileAssetIdentifier.getId()) || "folder".equals(fileAssetIdentifier.getAssetType()))){
+		if( null == fileAssetIdentifier || !InodeUtils.isSet(fileAssetIdentifier.getId()) || "folder".equals(fileAssetIdentifier.getAssetType())){
```

Comment:
I think you can use something like `Folder.getType()` to not use the String "folder" directly?